build.sbt

```diff
@@ -1262,9 +1261,7 @@ lazy val `scala-libs-wrapper` = project
         scalaLibrary ++
         scalaReflect ++
         slf4jApi ++
-        Seq(
-          "net.java.dev.jna" % "jna" % jnaVersion
-        ),
+        Seq(),
```

Comment:
nit: Remove empty Seq

Reply:
Gone since https://github.com/enso-org/enso/pull/13021#discussion_r2073307261 and merged into this PR then.